docs/paywall3.md

```diff
@@ -699,6 +699,7 @@ Here is the list of the HTML classes that can be used for creating each of the s
 | `inplayer-paywall-ccm`      | Invokes the  default card management screen |
 | `inplayer-paywall-change-pass`  | Invokes the 'Change Password' screen |
 | `inplayer-paywall-purchases`  | Invokes the 'My Purchases' paywall screen where an end-user can see the list of payments they have completed |
+| `inplayer-paywall-donations`	| Invokes the donations flow for the provided asset  |
```

Comment:
@liljanadimovska , let's discuss tomorrow if we should:

1. add another column in this table where we would document the ID
OR
2. leave a * at the end of the class and provide a more elaborate description of the usage below the table

Reply:
@matejpetrov updated